src/app/services/crud.service.ts

```diff
@@ -15,34 +15,32 @@ export class CrudService {
   private basePath = '/car';
```

Comment:
Ta basePath jest zła. Twoja collection w firebase nazywa się `cars` więc musisz zmienić na `/cars`
To jest powód, dlaczego nie pobierało Ci nic z bazy danych.